### ISSUE TITLE
Accessibility does not work 

### ISSUE THREAD

**Issue by Maxhc70** [SamSprung/SamSprung-TooUI#197]
### Issue Description

Accessibility does not work 

### Logcat Output

```shell
Accessibility does not work
```

### Wiki Confirmation

- [X] I have read and understood the current wiki

**Comment by AbandonedCart** [comment]
The log section is required when reporting an error.
